deploy/archive-sync.yaml

```diff
@@ -0,0 +1,397 @@
+              name: archive-sync-config
+          volumes:
+            - configMap:
+                name: archive-sync-config-map
+              name: archive-sync-config
+          resources:
+            requests:
+              cpu: ${CPU_REQUEST}
+              memory: ${MEMORY_REQUEST}
+            limits:
+              cpu: ${CPU_LIMIT}
+              memory: ${MEMORY_LIMIT}
+
+    kafkaTopics:
+      - replicas: 16
+        partitions: 3
+        topicName: ${KAFKA_INCOMING_TOPIC}
+      - replicas: 1
+        partitions: 1
+        topicName: ${KAFKA_DEAD_LETTER_QUEUE_TOPIC}
+      - replicas: 3
+        partitions: 1
+        topicName: ${KAFKA_OUTGOING_TOPIC}
+      - replicas: 3
+        partitions: 1
+        topicName: ${PAYLOAD_TRACKER_TOPIC}
+
+- kind: ConfigMap
+  apiVersion: v1
+  metadata:
+    labels:
+      app: archive-sync
+    name: archive-sync-config-map
+  data:
+    config.yaml: |-
+      plugins:
+        packages:
+        - ccx_messaging
+      service:
+        extract_timeout: ${EXTRACT_TIMEOUT}
+        extract_tmp_dir: ${EXTRACT_TMP_DIR}
+        format: insights.formats._json.JsonFormat
+        target_components: []
+        consumer:
+          name: ccx_messaging.consumers.idp_kafka_consumer.IDPConsumer
+          kwargs:
+              incoming_topic: ${KAFKA_INCOMING_TOPIC}
+              group.id: ${KAFKA_GROUP_ID}
+              bootstrap.servers: ${KAFKA_CONSUMER_SERVER}
+              security.protocol: ${KAFKA_CONSUMER_SECURITY_PROTOCOL}
+              sasl.mechanism: ${KAFKA_CONSUMER_SASL_MECHANISM}
+              sasl.username: ${KAFKA_CONSUMER_CLIENT_ID}   # TODO: How are these populated???
+              sasl.password: ${KAFKA_CONSUMER_CLIENT_SECRET}   # TODO: How are these populated???
+
+        downloader:
+          name: ccx_messaging.downloaders.s3_downloader.S3Downloader
+          kwargs:
+            access_key: ${SOURCE_S3_ACCESS_KEY}
+            secret_key: ${SOURCE_S3_SECRET_KEY}
+            endpoint_url: ${SOURCE_S3_URL}
+            bucket: ${SOURCE_S3_BUCKET}
+
+        engine:
+          name: ccx_messaging.engines.s3_upload_engine.S3UploadEngine
+          kwargs:
+            dest_bucket: ${TARGET_S3_BUCKET}
+            access_key: ${TARGET_S3_ACCESS_KEY}
+            secret_key: ${TARGET_S3_SECRET_KEY}
+            endpoint: ${TARGET_S3_URL}
+            archives_path_prefix: archives/compressed
+        publisher:
+          name: ccx_messaging.publishers.synced_archive_publisher.SyncedArchivePublisher
+          kwargs:
+            outgoing_topic: ${KAFKA_SYNCED_TOPIC}
+            bootstrap.servers: ${KAFKA_PUBLISHER_SERVER}
+            security.protocol: ${KAFKA_PRODUCER_SECURITY_PROTOCOL}
+            sasl.mechanism: ${KAFKA_PRODUCER_SASL_MECHANISM}
+            sasl.username: ${KAFKA_PRODUCER_CLIENT_ID}   # TODO: How are these populated???
+            sasl.password: ${KAFKA_PRODUCER_CLIENT_SECRET}   # TODO: How are these populated???
```

Comment:
Same as previous comment, populated from Clowder configuration, this parameters will be "ignored"